sklearn/ensemble/_stacking.py

```diff
@@ -483,9 +489,13 @@ def fit(self, X, y, sample_weight=None):
             Returns a fitted instance of estimator.
         """
         check_classification_targets(y)
-        self._le = LabelEncoder().fit(y)
-        self.classes_ = self._le.classes_
-        return super().fit(X, self._le.transform(y), sample_weight)
+        self._type_of_target = type_of_target(y)
+        self.classes_ = []
+        if self._type_of_target != "multilabel-indicator":
```

Comment:
We should as well set the `classes_` attribute in `"multilabel-indicator"`.
For instance, with a random forest we would get:

```python
from sklearn.datasets import make_multilabel_classification
from sklearn.ensemble import RandomForestClassifier
X, y = make_multilabel_classification()
model = RandomForestClassifier().fit(X, y)
model.classes_
```

```
[array([0, 1]), array([0, 1]), array([0, 1]), array([0, 1]), array([0, 1])]
```

So we should have the same behaviour here. We should also have the test align with this case.

Reply:
The best here might be to create a `LabelEncoder` for each column of `y`. Like this we can encode `y` in the meta-estimator and then decode the target `y`